### PR TITLE
changes in "code of conduct " section

### PR DESCRIPTION
**Pull request by annu12340**
image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -8738,4 +8738,73 @@ dt::last-of-type {
...
+.container1 .box {
+  width: 220px;
+  height: 200px;
+  border-radius: 10px;
+  background-color: white;
+  margin: 20px;
+  transition: 0.3s all ease-in-out;
+  position: relative;
+  font-family: "Dosis", sans-serif;
+  box-shadow: -10px 25px 50px rgba(0, 0, 0, 0.3);
```
Comment (iambk) [inline css/style.css]: try decreasing the box-shadow a bit.

**Review by iambk — CHANGES_REQUESTED** [review]
Also, can you make the thing more responsive? As of now the title cards hides away for smaller screen size.

File: css/style.css (hunk 1 above)
Comment (de-sh) [reply css/style.css]: And maybe make the on hover turn the borders darker instead of the current version...

**Comment by de-sh** [comment]
A lot of changes to be made, but pulling this still in. Please turn on GitHub pages so that we can check the effect of your changes in real time without having to download it.

**Pull request merged**